.neuro/live.yaml

```diff
@@ -0,0 +1,47 @@
+kind: live
+title: stable-diffusion-webui
+
+defaults:
+  life_span: 5d
+
+images:
+  stablediffusion:
+    ref: image:$[[ project.id ]]:v1
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
+    context: $[[ flow.workspace ]]/
+    build_preset: cpu-large
+
+volumes:
+  models:
+    remote: storage:$[[ flow.project_id ]]/models
+    mount: /home/worker/app/models
+    local: models
+  embeddings:
+    remote: storage:$[[ flow.project_id ]]/embeddings
+    mount: /home/worker/app/embeddings
+    local: embeddings
+
+jobs:
+  stablediffusioncpu:
+    image: ${{ images.stablediffusion.ref }}
```

Comment:
and use `:development` image here by default

Reply:
meant different

we could directly use our image from github (that we are about to publish), e.g. 
```
...
jobs:
  stablediffusioncpu:
    image: ghcr.io/neuro-inc/stable-diffusion-webui:development
...
```